plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -264,8 +264,6 @@ protected Optional<DataMappingTestSetup> filterDataMappingSmokeTestData(DataMapp
             case "time(6)":
             case "timestamp":
             case "timestamp(3)":
-            case "timestamp(3) with time zone":
```

Comment:
I don't think the connector supports creating the table with this type. Please take a look at `BigQueryType#toStandardSqlTypeName` method.

Reply:
I'll have a better look at this. tbh I couldn't understand very well how this is being tested. Can you expand on this? What's the expectation here?

Reply:
double checked and yeah this connector does not support the remove type.